src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_AMOUNT = 4;
+    private static final int MAX_VALUE_FOR_RANDOM_FIGURE = 50;
+    private final Random random = new Random();
+
+    public Figure getDefaultFigure() {
+        return new Circle(Color.WHITE, 10);
+    }
+
+    public Figure getRandomFigure() {
+        int randomIndex = random.nextInt(FIGURE_AMOUNT);
+        return new Figure[]{getRandomCircle(), getRandomRectangle(), getRandomIsoscelesTrapezoid(),
+                getRandomRightRectangle()}[randomIndex];
```

Comment:
better replace with switch-case block , it will make your code more optimized and faster